graphprot/ResidueGraph.py

```diff
@@ -253,8 +255,8 @@ def get_edge_features(self):
 
         for e in self.nx.edges:
             node1, node2 = e
-            self.nx.edges[node1, node2]['polarity'] = self._get_edge_polarity(
-                node1, node2)
+            #self.nx.edges[node1, node2]['polarity'] = self._get_edge_polarity(
+            #    node1, node2)
```

Comment:
so we don't have any edge features ? no polarity and no distance ? (sorry I forgot a bit about the inner workings of the code)